django/api/serializers/application_form.py

```diff
@@ -21,6 +24,14 @@ def get_serializer_context(self):
 
     def create(self, validated_data):
         request = self.context["request"]
+
+        drivers_licence = validated_data.get("drivers_licence")
+        if GoElectricRebate.objects.filter(drivers_licence=drivers_licence).exists():
```

Comment:
This should only be rejected on certain application statuses.

if the drivers licence exists, but the application status is in DECLINED, EXPIRED or NOT APPROVED, someone should be able to create a new application in this table.

If there's a way to make this a constraint at the database level that would be ideal.

Reply:
I think this is how Django achieves database contraints: https://docs.djangoproject.com/en/4.0/ref/models/constraints/

Reply:
This shouldn't be done in the serializer, see previous comments.